<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRegisterWidget</class>
 <widget class="QWidget" name="AddRegisterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>197</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0">
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="cmbConnection"/>
   </item>
   <item row="6" column="0">
    <widget class="QRadioButton" name="radioPrimary">
     <property name="text">
      <string>Y1 axis</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="lineName">
     <property name="text">
      <string>Name of curve</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QPushButton" name="btnAdd">
     <property name="text">
      <string>Add register</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QSpinBox" name="spinAddress">
     <property name="maximum">
      <number>105536</number>
     </property>
     <property name="value">
      <number>40001</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QComboBox" name="cmbType"/>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="radioSecondary">
     <property name="text">
      <string>Y2 axis</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineName</tabstop>
  <tabstop>spinAddress</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>cmbConnection</tabstop>
  <tabstop>radioPrimary</tabstop>
  <tabstop>radioSecondary</tabstop>
  <tabstop>btnAdd</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
